cmd/tapcli/assets.go

```diff
@@ -138,6 +139,21 @@ var mintAssetCommand = cli.Command{
 				"in order to avoid printing a large amount " +
 				"of data in case of large batches",
 		},
+		cli.StringFlag{
+			Name: "group_key_xpub",
```

Comment:
Do we also need to be able to specify optional leaves for the custom tapscript root?

Reply:
I would argue that we do eventually but not for the next release. So we could potentially save that for a later PR. What do you think?